### PR TITLE
Objectives

### PR DESCRIPTION
**Pull request by nikol-d97**

### CODE REVIEW

**Comment by sarah-ravioli** [comment]
looks good 🥇

File: doc/objectives.md [hunk 1]
```diff
@@ -1 +1,8 @@
+#Study Objectives
```
Comment (sarah-ravioli) [inline doc/objectives.md]: remove hashtag

**Review by sarah-ravioli — CHANGES_REQUESTED** [review]

File: doc/objectives.md [hunk 2]
```diff
@@ -1 +1,8 @@
+#Study Objectives 
+* What is the relationship between microinvertabrate community composition and seasonality? 
++ Prediction: community composition will vary with changing seasons
+- What is the relationship between water-body types, sites, community composition, and species richness? 
++ Prediction: community composition will be the same across sites with the same water-body type, and different acorss sites with different water-body types. 
+- What is the relationship between environmental conditions (pH, percipitation) across sites, community composition, and species richness? 
++ Changing pH levels will affect species community composition and abundances.
```
Comment (youngseojeong) [inline doc/objectives.md]: Add Prediction:

**Comment by YiranZhao** [comment]
Great!

**Pull request merged**